victor/src/main/groovy/com/trello/victor/VictorPlugin.groovy

```diff
@@ -83,15 +84,25 @@ class VictorPlugin implements Plugin<Project> {
                     return
                 }
 
-                Task rasterizationTask = project.task("rasterizeSvgsFor${variant.name.capitalize()}", type: RasterizeTask) {
-                    sources = svgFiles
-                    outputDir = project.file("$project.buildDir/generated/res/$flavorName/$buildType.name/svg/")
-                    includeDensities = densities
-                    baseDpi = project.victor.svgDpi
+
+                File conversionOutputDir = project.file("$project.buildDir/generated/res/$flavorName/$buildType.name/svg/")
```

Comment:
Wait, wouldn't that happen now since we're using the same directory for both tasks? My point is that if we switch based on output, then the user may have both an SVG->PNG and SVG->Android directory at the same time (so they can compare), but both won't be slurped up.